src/index.js

```diff
@@ -11,6 +11,13 @@ import assert from 'assert';
 import FusionApp from 'fusion-core';
 import type {FusionPlugin} from 'fusion-core';
 
+/* Note: as the Jest type definitions are declared globally and not as part of
+ * a module, we must import the relevant types directly from the libdef file here
+ * to avoid the invariant that all consumers must add the jest libdef to their
+ * .flowconfig libs.
+ */
+import type {JestTestName, JestObjectType} from './flow/jest_v22.x.x.js';
```

Comment:
Why not import this directly from the flow-typed directory instead of moving the file?

Reply:
Flow has problems resolving types from the `flow-typed` directory:

<img width="438" alt="screen shot 2018-04-10 at 2 56 52 pm" src="https://user-images.githubusercontent.com/3497835/38585646-725efb4c-3ccf-11e8-8112-ed1afceebbcc.png">